### PR TITLE
Update the assessment of the codebase to the Standard for Public Code v0.2.2

### LINKED ISSUE #1087: Update the Standard for Public Code assessment to v0.2.2
There is a new version of the Standard, so [the assessment](https://github.com/open-zaak/open-zaak/blob/main/docs/introduction/open-source/public_code.md) could be updated.

### PR DESCRIPTION
**Pull request by ericherman**
Fixes #1087

**Changes**

Updated `docs/introduction/open-source/public_code.md` to match the [current v0.2.2 template](https://github.com/publiccodenet/about/blob/develop/activities/codebase-auditing/review-template.md)

Co-authored-by: Jan Ainali <ainali.jan@gmail.com>

### CODE REVIEW

File: docs/introduction/open-source/public_code.md [hunk 1]
```diff
@@ -114,21 +116,22 @@ Requirement | meets | links and notes
...
 The documentation of the codebase SHOULD contain: a high level description that is clearly understandable for a wide audience of stakeholders, like the general public and journalists, a section describing how to install and run a standalone version of the source code, including, if necessary, a test dataset, examples for all functionality. |  | [documentation](https://open-zaak.readthedocs.io/en/latest/introduction/index.html), would like to see standard recipies and examples of more fuctionality
-There SHOULD be continuous integration tests for the quality of your documentation. | yes | link checks, build checks, [Travis config](https://github.com/open-zaak/open-zaak/blob/master/.travis.yml#L74)
+There SHOULD be continuous integration tests for the quality of the documentation. | yes | link checks, build checks, [Travis config](https://github.com/open-zaak/open-zaak/blob/master/.travis.yml#L74)
```
Comment (sergei-maertens) [inline docs/introduction/open-source/public_code.md:119]: Link should be updated to the Github Actions, as we migrated from Travis to Github Actions.

**Review by sergei-maertens — APPROVED** [review]

File: docs/introduction/open-source/public_code.md [hunk 2]
```diff
@@ -70,13 +70,15 @@ The documentation SHOULD include instructions for how to report potentially secu
...
-You SHOULD group relevant changes in commits. | yes | [PRs](https://github.com/open-zaak/open-zaak/pulls)
-You SHOULD mark different versions of the codebase, for example using revision tags or textual labels. | yes | [releases](https://github.com/open-zaak/open-zaak/releases)
-You SHOULD prefer file formats that can easily be version controlled. | yes | mostly code and Restructured Text or Markdown
+The codebase SHOULD be maintained in a distributed version control system. | yes | git
+Contributors SHOULD group relevant changes in commits. | yes | [PRs](https://github.com/open-zaak/open-zaak/pulls)
+Maintainers SHOULD mark released versions of the codebase, for example using revision tags or textual labels. | yes | [releases](https://github.com/open-zaak/open-zaak/releases)
+Contributors SHOULD prefer file formats where the changes within the files can be easily viewed and understood in the version control system. | yes | mostly code and Restructured Text or Markdown
+Contributors MAY sign their commits and provide an email address, so that future contributors are able to contact past contributors with questions about their work. |  |
```
[the remark this answers is not in the document]
Comment (ericherman) [reply docs/introduction/open-source/public_code.md]: Added.

File: docs/introduction/open-source/public_code.md (hunk 1 above)
Comment (ericherman) [reply docs/introduction/open-source/public_code.md]: Added.

**Comment by ericherman** [comment]
Would you like me to squash to a single commit?

**Comment by sergei-maertens** [comment]
> Would you like me to squash to a single commit?

If you prefer, but I don't have any particular strong feelings about it :-)

**Pull request merged**